components/Layout.tsx

```diff
@@ -59,16 +42,21 @@ const Layout: React.FC = ({ children }) => {
       </Head>
       <div>
         <NavigationView>
-          <NavigationColumnLayout>
-            <NavigationHeaderLayout>
-              {walletAddress && client && <NewMessageButton />}
-            </NavigationHeaderLayout>
-            <NavigationPanel onConnect={handleConnect} />
-            <UserMenu
-              onConnect={handleConnect}
-              onDisconnect={handleDisconnect}
-            />
-          </NavigationColumnLayout>
+          <aside className="flex w-full md:w-84 flex-col flex-grow fixed inset-y-0">
```

Comment:
I'm not sure what this has to do with upgrading to React 18

Reply:
Umm I removed this as every component requires children prop explicitly in React 18, I saw no reason to have components to pass children around and cause reloads